platform/cavium/cavm_platform_modules/Makefile

```diff
@@ -0,0 +1,27 @@
+.ONESHELL:
+SHELL = /bin/bash
+.SHELLFLAGS += -e
+
+MAIN_TARGET = cavm_platform_modules.deb
+DEB_BUILD_DIR = cavm-platform-modules-deb
+KVERSION = linux-3.16.36
+
+$(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
+	# get sources
+	rm -rf SONiC
+	git clone https://github.com/edge-core/SONiC.git
+
+	# build
+	pushd SONiC/AS7512-32X/module/
+
+	KERNEL_SRC=/sonic/src/sonic-linux-kernel/$(KVERSION)/debian/build/build_amd64_none_amd64 make
```

Comment:
I don't think that using a path in sources is a good idea
It is obtained by fetching archives from online repositories during linux kernel build
In case these modules are built before linux kernel, build will fail